app_base.htm

```diff
@@ -0,0 +1,70 @@
+## APP BASE CONTAINS THE NAV BAR
+
+<%inherit file="/homepage/templates/base.htm" />
+import
+
+<%! from catalog import models as cmod %>
+
+##set the active tab dynamically with code
+<%block name="menu">
+<ul class="nav navbar-nav navbar-left">
+  <li class="${'active' if request.dmp.page == 'about' else ''}"><a href="homepage/about">About</a></li>
+    <li class="${'active' if request.dmp.page == 'contact' else ''}"><a href="homepage/contact">Contact</a></li>
+    <li class="${'active' if request.dmp.page == 'terms' else ''}"><a href="homepage/terms">Terms</a></li>
+    <li class="${'active' if request.dmp.page == 'faq' else ''}"><a href="homepage/faq">FAQ</a></li>
+  <li class="${'active' if request.dmp.page == 'sections' else ''}"><a href="homepage/sections">Sections</a></li>
+</ul>
+
+<ul class="nav navbar-nav navbar-right">
+  %if request.user.is_authenticated:
+  <div class="dropdown navbar-right">
+    <button class="btn btn-primary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+      ${request.user.first_name}
+        &#9660;
```

Comment:
I'm unsure what the &#9660 is meant to do

Reply:
it imports a "down arrow" icon